lib/src/clixon_event.c

```diff
@@ -53,6 +53,7 @@
 #include <sys/param.h>
 #include <sys/types.h>
 #include <sys/time.h>
+#include <poll.h>
```

Comment:
Can you please add a compile-time constant around the poll change, so we can introduce it gradually, and have it as experimental code for a while
I would suggest :
`#ifdef CLIXON_EVENT_POLL`
and define the constant in clixon_custom.h

Reply:
> and define the constant in clixon_custom.h

@olofhagsand I didn't quite understand why clixon_custom.h is needed.

Reply:
> Can you please add a compile-time constant around the poll change, so we can introduce it gradually, and have it as experimental code for a while
> I would suggest :
> `#ifdef CLIXON_EVENT_POLL`

@olofhagsand I did this `./configure --enable-event-poll`

Reply:
> Can you please add a compile-time constant around the poll change, so we can introduce it gradually, and have it as experimental code for a while I would suggest : `#ifdef CLIXON_EVENT_POLL` and define the constant in clixon_custom.h

I think I got it, thanks :)

Reply:
> > and define the constant in clixon_custom.h
> 
> @olofhagsand I didn't quite understand why clixon_custom.h is needed.

It is not needed when you added it in configure.ac, so it can be removed

Reply:
> > Can you please add a compile-time constant around the poll change, so we can introduce it gradually, and have it as experimental code for a while
> > I would suggest :
> > `#ifdef CLIXON_EVENT_POLL`
> 
> @olofhagsand I did this `./configure --enable-event-poll`

When you add this to configure.ac, you also need to run "autoheader" and then add include/clixon_config.h.in.
Then you dont need the clixon_custom.h part.